versioned_docs/version-1.0.0/migration/installation.md

```diff
@@ -1,92 +1,86 @@
 ---
 id: install
 title: Install  Netcode for GameObjects
-description: Install the on-going development version of Unity  Netcode for GameObjects from a cloned respository. 
+description: Install 1.0.0 version of Unity  Netcode for GameObjects. 
 ---
 
-Multiple methods are available to install the  Netcode for GameObjects (Netcode) package. This guide details how to install the [**develop** branch](https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi/tree/develop) of Netcode.
+This guide details how to install Unity  Netcode for GameObjects (Netcode) via the Unity Hub.
 
-:::important In-Progress Development
-All features and code available in the *develop* branch is in-progress and not final. All cloned code may change daily or weekly, depending on submitted pull requests. All documentation and release notes for the *develop* version are not final.
-:::
 
-## Clone the repo
 
-You need to clone the code repo locally to work with in-development code.
 
-First, install [Git](https://git-scm.com/) if you do not have it installed on your PC. After installing Git, restart your system. A full restart is required to update for Git or you may receive an error adding packages.
+This guide assumes that you already have:
+  * Git installed on your PC. If you don't already have it, download [Git](https://git-scm.com/) and restart your system. A full restart is required to update Git or you may receive an error adding packages.
```

Comment:
```suggestion
```
This should no longer be necessary since we fetch directly from the Unity package registry.